reporters/kamon-opentelemetry/src/main/resources/reference.conf

```diff
@@ -14,25 +14,64 @@ kamon.otel.trace {
   # https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md
   endpoint = ${kamon.otel.trace.protocol}"://"${kamon.otel.trace.host}":"${kamon.otel.trace.port}
   endpoint = ${?OTEL_EXPORTER_OTLP_ENDPOINT}
-  endpoint = ${?OTEL_EXPORTER_OTLP_TRACES_ENDPOINT}
 
-  # Enable or disable including tags from kamon.environment.tags as resource labels in the exported trace
-  # Any keys containing a hyphen (-) will be converted to a dot (.) as that is the standard.
-  # e.g. service-version becomes service.version
-  # reference: https://github.com/kamon-io/Kamon/blob/master/core/kamon-core/src/main/resources/reference.conf#L23
+  # Supports empty string or gzip
+  compression = ""
+  compression = ${?OTEL_EXPORTER_OTLP_COMPRESSION}
+
+  # Supports comma-separated pairs (i.e "api-key=supersecret,data-type=application-traces")
+  headers = ""
+  headers = ${?OTEL_EXPORTER_OTLP_HEADERS}
+
+  timeout = 10s
+  timeout = ${?OTEL_EXPORTER_OTLP_TIMEOUT}
+
+  # Supports grpc and http/protobuf
+  otelProtocol = "grpc"
+  otelProtocol = ${?OTEL_EXPORTER_OTLP_PROTOCOL}
+
   include-environment-tags = no
+  trace {
+      host = ${kamon.otel.host}
+      port = ${kamon.otel.port}
+      protocol = ${kamon.otel.protocol}
+
+      endpoint = ${kamon.otel.endpoint}
+      fullEndpoint = ${?OTEL_EXPORTER_OTLP_TRACES_ENDPOINT}
+
+      compression = ${kamon.otel.compression}
+      compression = ${?OTEL_EXPORTER_OTLP_TRACES_COMPRESSION}
 
-  # Arbitrary key-value pairs that further identify the environment where this service instance is running.
-  # These are added as KeyValue labels to the Resource part of the exported traces
-  # Requires 'include-environment-tags' to be set to 'yes'
-  #
-  # kamon.environment.tags {
-  #   service-version = "x.x.x"
-  #   service-namespace = "ns"
-  #   service-instance.id = "xxx-yyy"
-  # }
+      headers = ${kamon.otel.headers}
+      headers = ${?OTEL_EXPORTER_OTLP_TRACES_HEADERS}
+
+      timeout = ${kamon.otel.timeout}
+      timeout = ${?OTEL_EXPORTER_OTLP_TRACES_TIMEOUT}
+
+      otelProtocol = ${kamon.otel.otelProtocol}
+      otelProtocol = ${?OTEL_EXPORTER_OTLP_TRACES_PROTOCOL}
+
+      # Enable or disable including tags from kamon.environment.tags as resource labels in the exported trace
+      # Any keys containing a hyphen (-) will be converted to a dot (.) as that is the standard.
+      # e.g. service-version becomes service.version
+      # reference: https://github.com/kamon-io/Kamon/blob/master/core/kamon-core/src/main/resources/reference.conf#L23
+      include-environment-tags = ${kamon.otel.include-environment-tags}
```

Comment:
I'm not really sure this should be configurable. We have this setting in other reporters because there was never a place to put what we call "environment", but in OTel that place is definitely the Resource. I think all environment tags should go into the resource all the time, and additionally include:
  - `service.instance.id` with the value of instance in the environment
  - `host.name` with the value of host in the environment